tests/autorestart_test/test_container_autorestart.py

```diff
@@ -0,0 +1,101 @@
+"""
+Check the auto-restart feature of syncd, swss, teamd, bgp and dhcp_relay
+"""
+import time
+import logging
+
+from common.utilities import wait_until
+
+CONTAINER_STOP_TEST_WAITING_TIME = 30
+CONTAINER_STOP_TEST_CHECK_INTERVAL = 1
+CONTAINER_RESTART_TEST_WAITING_TIME = 180
+CONTAINER_RESTART_TEST_CHECK_INTERVAL = 1
+CONTAINER_RUNNING_TEST_WAITING_TIME = 30
+CONTAINER_RUNNING_TEST_CHECK_INTERVAL = 1
```

Comment:
Why do we need these? They are the same values we use for testing that the container has stopped. We just use them in an inverse way -- we WANT the threshold to be reached in this case. I suggest removing these and using the same constants in both tests. This way, if we ever change the values, we only have to change them in one place.